dsa-solutions/lc-solutions/0000-0099/0020-valid.parentheses.md

```diff
@@ -0,0 +1,140 @@
+---
+id: valid-parentheses
+title: Valid Parentheses (LeetCode)
+sidebar_label: 0020-valid-parentheses
+tags:
+  - Stack
+
+description: "Given a string s containing just the characters '(', ')', '{', '}', '[' and ']', determine if the input string is valid."
+---
+
+## Problem Description
+
+| Problem Statement                                                                  | Solution Link                                                                                 | LeetCode Profile                                         |
+| :--------------------------------------------------------------------------------- | :-------------------------------------------------------------------------------------------- | :------------------------------------------------------- |
+| [Valid Parentheses ](https://leetcode.com/problems/valid-parentheses/description/) | [Valid Parentheses ( on LeetCode](https://leetcode.com/problems/valid-parentheses/solutions/) | [Vikas Pathak](https://leetcode.com/u/Vikas-Pathak-123/) |
+
+### Problem Description
+
+Given a string s containing just the characters '(', ')', '{', '}', '[' and ']', determine if the input string is valid.
+
+An input string is valid if:
+
+1. Open brackets must be closed by the same type of brackets.
+2. Open brackets must be closed in the correct order.
+3. Every close bracket has a corresponding open bracket of the same type.
+
+### Examples
+
+#### Example 1
+
+- **Input:** `s = "()"`
+- **Output:** true
+
+#### Example 2
+
+- **Input:** `s = "()[]{}"`
+- **Output:** true
+
+#### Example 3
+
+- **Input:** `s = "(]"`
+- **Output:** false
+
+### Constraints
+
+- $1 <= s.length <= 104$
+- $s consists of parentheses only '()[]{}'.$
```

Comment:
replace `- $s consists of parentheses only '()[]{}'.$` to `- s consists of parentheses only '()[]{}'.`